aerosol_dynamics/particle.py

```diff
@@ -0,0 +1,184 @@
+        return self._mass
+
+    @u.wraps(u.m, [None])
+    def radius(self) -> float:
+        """Returns radius of a particle. Checks units. [m]"""
+        return self._radius
+
+    @u.wraps(u.dimensionless, [None])
+    def charge(self) -> float:
+        """Returns charge of a particle. Checks units. [unitless]"""
+        return self._charge
+
+    @u.wraps(u.dimensionless, [None])
+    def knudsen_number(self) -> float:
+        """Returns a particle's Knudsen number. Unitless.
+
+        The Knudsen number seeks to reflect the relative length scales of the
+        particle and the suspending fluid (air, water, etc.). This is calculated
+        by the mean free path of the medium divided by the particle radius.
+        """
+        return pp.MEAN_FREE_PATH_AIR / self.radius()
+
+    @u.wraps(u.dimensionless, [None])
+    def slip_correction_factor(self) -> float:
+        """Returns a particle's Cunningham slip correction factor. Unitless.
+
+        The slip correction factor is a dimensionless quantity that accounts for
+        non-continuum effects when calculating the drag on small particles.
+        This is a deviation from Stokes' Law; Stokes assumes a no-slip
+        condition that is no longer correct at high Knudsen numbers.
+        The slip correction factor is used to calculate the friction factor.
+        See Eq 9.34 in Atmos. Chem. & Phys. (2016) for more information.
+        """
+        knudsen_number: float = self.knudsen_number()
+        return 1 + knudsen_number * (1.257 + 0.4*np.exp(-1.1/knudsen_number))
+
+    @u.wraps(u.kg / u.s, [None])
+    def friction_factor(self) -> float:
+        """Returns a particle's friction factor. [N-s/m].
+
+        The friction factor is a property of the particle's size and the medium
+        that the particle is in. Multiplying the friction factor by the fluid
+        velocity gives the drag force on the particle.
+        """
+        slip_correction_factor: float = self.slip_correction_factor()
+        return 6 * np.pi * pp.MEDIUM_VISCOSITY * self.radius() / slip_correction_factor
+
+    @u.wraps(u.kg, [None, None])
+    def reduced_mass(self, other) -> float:
+        """Returns the reduced mass of two particles. [kg].
+
+        The reduced mass is an "effective inertial" mass that allows a two body
+        problem to be solved as a one body problem.
+        """
+        return self.mass() * other.mass() / (self.mass() + other.mass())
+
+    def reduced_friction_factor(self, other) -> float:
+        """Returns the reduced friction factor between two particles. [N-s/m]
+       
+        Similar to the reduced mass, the reduced friction factor allows a two
+        body problem to be solved as a one body problem.
+        """
+        return (self.friction_factor() * other.friction_factor()
+            / (self.friction_factor() + other.friction_factor()))
+
+    def coulomb_potential_ratio(self, other) -> float:
+        """
+        """
```

Comment:
are these in S&P, or do we need another citation for these charge equations.

Reply:
We don't strictly speaking need another citation for these things. I can later do a thorough documentation of all equations step-by-step. These things are sort of well known --- a simplistic treatment along these lines does exists in S&P (I believe in he appendices). However, this particular treatment takes a detour into the dimensionless direction --- quite different than classic methods, but still well-documented and studied in the literature. In the future, I can add other alternative approaches as backup or options.

Reply:
I think it would be good to have references for where each equation comes from. Even if it's well-documented in the literature, I would hate for a new student (or even me, who never worked with charges) to have to dig for papers to convince themselves it's correct or understand the context for how it's derived and used. I couldn't find this exact equation in S&P or through a cursory search for "coulomb potential ratio", so having it explicitly pointed out would be tremendously helpful.

Reply:
On "coulomb potential ratio" there is a high chance I just made that up to make the whole logic flow better. I will submit an issue now to add a more thorough documentation of these steps.